### PR TITLE
get on that good retry patterns for checking to see if a legistar web url is valide

### PR DESCRIPTION
**Pull request by fgregg**
In the quest to squash all 104s, this is the current challenge. Depends on https://github.com/opencivicdata/python-legistar-scraper/pull/88

### CODE REVIEW

File: lametro/events.py [hunk 1]
```diff
@@ -340,6 +341,11 @@ class LAMetroWebEvent(dict):
     This class is for adding methods to the web event dict
     to facilitate labeling and sourcing audio appropriately.
     '''
+
+    web_scraper = LegistarScraper(retry_attempts=3,
+                                  requests_per_minute=0)
+    web_scraper.params = {}
```
Comment (hancush) [inline lametro/events.py]: do we need to set this? looks like the scrapelib session inherits from `requests.Session`, which [defaults `params` to an empty dict](https://github.com/kennethreitz/requests/blob/d22d219d34c81a1f5e599a43b3beb9449c5cf15e/requests/sessions.py#L387), and i don't see that it's changed in any of the scrapelib session classes or the legistar scraper.

Comment (fgregg) [reply lametro/events.py]: we don't need now. this is a bit of defensive programming, but maybe don't need it yet.

Comment (fgregg) [reply lametro/events.py]: let me know whether you think we should keep. I'm on the fence.

Comment (hancush) [reply lametro/events.py]: can you say about more about what you're defending against? 

it seems like we're hedging against params being set on the base legistar scraper class in the `python-legistar` lib. design wise, is that something we'd do? i can't think of an instance where there would be a unified param need for both the web and api child classes, i.e., i'd think any scraper level param setting would happen on those child classes [or downstream](https://github.com/datamade/scrapers-us-municipal/blob/92b07b08c391f42cee9548fcc5b70bf946e143fc/lametro/bills.py#L26-L47), for jurisdiction-specific use cases. still, i've looked past obvious cases before! :-) 

tl;dr – if what we're defending against is very unlikely, i'd strike this line; otherwise, i'm ok with keeping it!
